packages/google_maps_flutter/lib/src/controller.dart

```diff
@@ -236,4 +236,15 @@ class GoogleMapController {
         'map#getLatLng', screenCoordinate._toJson());
     return LatLng(latLng[0], latLng[1]);
   }
+
+  /// Returns [double] corresponding to the distance in meters of the [LatLng] in the current map view.
+  ///
+  /// Return NaN (Not-a-Number) if unable to get distance in meters.
+  /// This is only accurate for small Earth distances.
+  Future<double> getDistance(double meters, LatLng latLng) async {
```

Comment:
The iOS Google Map SDK says "Converts a distance in meters to content size."
Maybe we should say something similar in our dart doc to avoid confusion?

What happens if meters is null, or if latlng is null?

Reply:
The wording for dart docs have been rephrased as you suggested above, also we have added assertions for meters and latlng to be non null values [here](packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart)